src/index.js

```diff
@@ -9,6 +9,7 @@ import store from './state/store/configureStore';
 import { BrowserRouter as Router } from 'react-router-dom';
 
 axios.defaults.baseURL = 'https://fake-newzzzz.herokuapp.com/api';
+// axios.defaults.baseURL = 'http://localhost:3000/api';
```

Comment:
Why is it a comment here?

Reply:
Some times we want to test how it connects to real API on localhost to see if all works, so it is handy to have this line commented for this case. It will be removed when project is finished

Reply:
okey! 👍